os/arch/arm/src/stm32/stm32_otgfsdev.c

```diff
@@ -2740,14 +2783,14 @@ static inline void stm32_epin_interrupt(FAR struct stm32_usbdev_s *priv)
 			 */
 #if 0
 			if ((diepint & OTGFS_DIEPINT_EPDISD) != 0) {
-				usbtrace(TRACE_INTDECODE(STM32_TRACEINTID_EPIN_EPDISD), (uint16_t)diepint);
+				usbtrace(TRACE_INTDECODE(STM32_TRACEINTID_EPIN_EPDISD), (uint16_t) diepint);
 				stm32_putreg(OTGFS_DIEPINT_EPDISD, STM32_OTGFS_DIEPINT(epno));
 			}
 #endif
 			/* Transmit FIFO empty */
 
 			if ((diepint & OTGFS_DIEPINT_TXFE) != 0) {
-				usbtrace(TRACE_INTDECODE(STM32_TRACEINTID_EPIN_TXFE), (uint16_t)diepint);
+				usbtrace(TRACE_INTDECODE(STM32_TRACEINTID_EPIN_TXFE), (uint16_t) diepint);
```

Comment:
Could you check all of this kind of change?

Reply:
All such changes has been taken care now